R/count_teleconnections.R

```diff
@@ -133,6 +137,21 @@ count_watershed_teleconnections <- function(data_dir,
 
         distance_values %>% mean()/m_to_km -> avg_withdr_dis_km
 
+        # Get number of cities
+        watershed_usage_table %>%
+          subset(Watershed_DVSN_ID %in% city_intake_ids) -> city_usage
+        if(nrow(city_usage) == 0){
+          dependent_population <- 0
+          n_other_cities <- 0
+        }else{
+        city_usage[,4] %>% unique() %>%
```

Comment:
please name the column for clarity (as opposed to having "4"). You can use something like `dplyr::select(city_state)`